react/features/base/connection/reducer.js

```diff
@@ -188,3 +196,16 @@ function _setLocationURL(
         options: locationURL ? _constructOptions(locationURL) : undefined
     });
 }
+
+/**
+ * Reduces a specific redux action {@link SET_ROOM} of the feature
+ * base/connection.
+ *
+ * @param {Object} state - The redux state of the feature base/connection.
+ * @private
+ * @returns {Object} The new state of the feature base/connection after the
+ * reduction of the specified action.
+ */
+function _setRoom(state: Object) {
+    return set(state, 'error', undefined);
```

Comment:
@lyubomir I went for this, but I'm not happy about it. We need a way to clean the error state when we either rejoin or cancel. I was thinking about some action dispatched *before* we load the config. WDYT? Any name suggestions?